src/Fantomas.Tests/CommentTests.fs

```diff
@@ -43,8 +43,8 @@ let print_30_permut() =
     /// declare and initialize
     let permutation: int array =
         Array.init n (fun i ->
-            Console.Write(i + 1)
-            i)
+                Console.Write(i + 1)
```

Comment:
IMO we don't want double indentation here

Reply:
Reverted